variables.tf

```
@@ -0,0 +1,41 @@
+variable "chart_version" {
+  description = "Chart version of the ingress controller"
+  type        = string
+}
+
+variable "namespace" {
+  description = "Namespace of the ingress controller"
+  type        = string
+}
+
+variable "host" {
+  description = "The hostname (in form of URI) of Kubernetes master"
+  type        = string
+}
+
+variable "cluster_ca_certificate" {
+  description = "PEM-encoded root certificates bundle for TLS authentication"
+  type        = string
+}
+
+variable "token" {
+  description = "Token to authenticate a service account"
+  type        = string
+}
+
+variable "lifecycle_ignore_changes" {
+  description = "Ignore lifecycle changes"
+  type        = list
+}
+
+variable "additional_values" {
+  description = "Additional values to pass to the helm chart"
+  type        = any
+  default     = {}
+}
+
+variable "ssl_reject_handshake" {
+  description = "Reject SSL handshake if it does not support SNI"
```

I would prefer something like:

```suggestion
  description = "Reject SSL handshake if the client does not ask for a configured domain"
```

what do you think?